### ISSUE TITLE
Remote Path Mapping not working

### ISSUE THREAD

**Issue by BigJazzz** [Sonarr/Sonarr#4358]
When a downloaded episode is available for import, it's not being imported automatically. When I view the Activity -> Queue page, I see this error when hovering over the orange download icon:
`The.Boys.S02E07.Butcher.Baker.Candlestick.Maker.720p.AMZN.WEB-DL.DDP5.1.H.264-NTb
[\\MediaNAS\Media\complete\TV\The.Boys.S02E07.Butcher.Baker.Candlestick.Maker.720p.AMZN.WEB-DL.DDP5.1.H.264-NTb\] is not a valid local path. You may need a Remote Path Mapping.`
Looking at the additional information (using the person icon) I get:
`value [\\MediaNAS\Media\complete\TV\The.Boys.S02E07.Butcher.Baker.Candlestick.Maker.720p.AMZN.WEB-DL.DDP5.1.H.264-NTb\] is not a valid *nix path. paths must start with / Parameter name: path`
Manually importing works just fine, and I have the exact same set up in Radarr and have no issues there. This was working correctly before upgrading to v3.

**To Reproduce**
1. Send an episode to SAB for download
2. When download is finished, visit Activity -> Queue and notice the orange download icon
3. Hover over the icon
4. See error (mentioned above)

**Expected behavior**
Remote Port Mapping settings to be recognised and utilised, and imports to happen automatically.

**Screenshots**
![Untitled](https://user-images.githubusercontent.com/2503113/110049505-124cd380-7da6-11eb-86c5-2035b7f8b95b.png)
![Untitled](https://user-images.githubusercontent.com/2503113/110049812-b767ac00-7da6-11eb-8442-062e39273050.png)

**Platform Information (please complete the following information):**
 - OS: Synology NAS DS918+, DSM version: DSM 6.2.3-25426 Update 3
 - Docker: No
 - .net Framework (Windows) or mono (macOS/Linux) (System -> Status): 5.20.1.34-16
 - Sonarr Version: 3.0.4.1132
 - Sonarr Branch: phantom-develop

**Trace Logs**
[sonarr.trace.txt](https://github.com/Sonarr/Sonarr/files/6087394/sonarr.trace.txt)

**Comment by markus101** [comment]
Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC](http://webchat.freenode.net/?channels=#sonarr) for support/questions.

**Comment by BigJazzz** [comment]
I'm not entirely sure how this isn't a bug, considering it should be working, was working, and now isn't working with an update.

**Comment by markus101** [comment]
The wrong host or the wrong remote path accounts for almost every “bug” with remote path mappings. If the host matches the download client then it will try to replace the remote path with the local path, something doesn’t match in your setup.

**Comment by BigJazzz** [comment]
As I said originally, Radarr works perfectly. It's only been since I upgraded Sonarr. As you can see, the Remote Path Mappings are almost identical, it's just the final folder that is different (for obvious reasons). These are the exact same settings I had before I upgraded Sonarr. Again, I haven't changed anything, again it was working before the upgrade.
![image](https://user-images.githubusercontent.com/2503113/110057509-d40ae080-7db4-11eb-9301-5d64d50ad113.png)
![image](https://user-images.githubusercontent.com/2503113/110057523-dbca8500-7db4-11eb-93b8-0790535301f6.png)

**Comment by bakerboy448** [comment]
Well then providing actual trace logs would be the first step to solving anything, which would be discussed via our support channels.

I’m not sure if you uploaded the wrong file to the OP or don’t know how to get logs, wiki has links either way

**Comment by BigJazzz** [comment]
Ah, I uploaded the wrong file. My apologies. I think that should be the correct file now.
[sonarr.trace.txt](https://github.com/Sonarr/Sonarr/files/6087745/sonarr.trace.txt)

**Comment by bakerboy448** [comment]
1) you’re lying about uploading trace logs. You took the info log file, renamed it, and uploaded it.
2) your logs indicate it cannot connect to your download client; thus the remote map isn’t even in the picture 

`NzbDrone.Core.Download.Clients.DownloadClientUnavailableException: Unable to connect to SABnzbd, Error: ConnectFailure (Connection refused): 'http://192.168.1.2:8080/api?mode=queue&start=0&limit=0&apikey=(removed)&output=json' ---> System.Net.WebException: Error: ConnectFailure (Connection refused): 'http://192.168.1.2:8080/api?mode=queue&start=0&limit=0&apikey=(removed)&output=json' --->  `
3) the logs show a manual import with the incorrect path input by the user.

there is no known bug with remote map.
Your logs are not trace as requested and are in fact renamed info. 


please see our troubleshooting article on the wiki https://wiki.servarr.com/Sonarr_Troubleshooting

if you have further questions and need further assistance use the appropriate support channels.  GitHub is not for support and frankly there are very few “bug” issues opened that are actually bugs, most...like yours... are support due to improper configuration by the user.

Additionally, as GitHub states exceptions are not bugs.

Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC ](http://webchat.freenode.net/?channels=#sonarr)for support/questions.

please do not continue to comment on what appears to be an improperly filed bug support and use the appropriate support avenues.

**Comment by BigJazzz** [comment]
1. Wow. No need to be so rude. I made an honest mistake. I get the frustration that comes from providing help, but that doesn't excuse the attitude. Maybe next time confirm which file someone meant to upload, as they may have acted in good faith.
2. This is news to me, as it's connecting just fine when I submit a download request and when it updates the Activity page. Sonarr is throwing no errors about not being able to connect to SAB in the Download Client area, and the test passes just fine. (Proof the test passes fine.)
![image](https://user-images.githubusercontent.com/2503113/110059273-0702a380-7db8-11eb-8dd4-c59485a71b04.png)
3. Yes, I *did* say that manually import the files just fine, though I'm not sure what you mean by "a manual import with the incorrect path input by the user". The path I put into the manual import page is the *exact* same path as what has been mapped.
4. Yes, no *known* bugs.
5. I understand the "improper configuration" argument. I feel like now you're just choosing to ignore everything I have said, especially the part where I say "I have changed nothing since upgrading, and it was working as expected before upgrading". 
[sonarr.trace.txt](https://github.com/Sonarr/Sonarr/files/6087781/sonarr.trace.txt)

If this was a fresh install and configuration, then I would be inclined to agree with you I may have misconfigured something. It's not, I haven't changed anything, Sonarr is not throwing SAB connection errors, and the exact same configuration works in Radarr.

**Comment by bakerboy448** [comment]
You're right it is very frustrating when nearly every single person seeking support today failed to read the instructions provided to them in plain english to facilitate them receiving support.  Surprisingly, users who follow the instructions, use the appropriate channels, and provide the appropriate information when requested tend to solve their issue significantly faster, which no frustration on either party's end, and with high satisfaction rates.  However, you are doing the very opposite every step of the way.

Let me make this short and sweet since it appears you skim over replies...

- The only possible way a file named sonarr.trace.txt contains no trace logs is deliberate action by the user, twice.
- The provided trace logs do not indicate any error nor any attempted import.
- Per our troubleshooting article you were linked and either failed to read or just ignored, the proper steps to provide useful trace logs 

> If you're linked here for support remember to get them from the actual log file, put the logs in a pastebin and show us context around what we need to see.
> 
> What you need to do is:
> 
> Turn Logging up to Trace
> Clear Logs
> Reproduce the Issue
> Open the trace log file (Sonarr.trace.txt)
> Unix Only: Alternatively If you're looking for a specific entry in an old log file but aren't sure which one you can use grep. For example if you want to find information about the movie or show "Shooter" you can run the following command grep -inr -C 100 -e 'Shooter' /path/to/logs/*.trace*.txt If your Appdata Directory is in your home folder then you'd run: grep -inr -C 100 -e 'Shooter' /home/$User/.config/logs/*.trace*.txt
> The flags have the following functions
> -i: ignore case
> -n: show line number
> -r: recursively check all files in the path
> -C: provide # of lines before and after the line it is found on
> -e: the pattern to search for
> Hastebin, Pastebin, 0bin, or any other pastebin type site **a big chunk before the issue, the issue, and a big chunk after the issue.**


- ### Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC ](http://webchat.freenode.net/?channels=#sonarr)for support/questions.

- You've now been asked no less than five times to use the proper channels. I strongly encourage you to do so.
-[Support Requests will be closed immediately, if you are not 100% certain this is a bug please go to our Reddit, Discord, Forums, or IRC first. Exceptions do not mean you found a bug!](https://github.com/Sonarr/Sonarr/issues/new/choose)
-[\](https://github.com/Sonarr/Sonarr/issues/new?assignees=&labels=&template=bug_report.md&title=)
-https://github.com/Sonarr/Sonarr/issues/4358#issuecomment-791056495
-https://github.com/Sonarr/Sonarr/issues/4358#issuecomment-791099404
